### PR TITLE
Connection to API and front end to back end

### PR DESCRIPTION
**Pull request by wynnc**

### CODE REVIEW

File: client/src/pages/Profile/Profile.js [hunk 1]
```diff
@@ -61,13 +89,51 @@ class Profile extends Component {
                                 <Link className="loginLink" to="/login"><Button className="loginBtn">Login</Button></Link>
                             </>
                         ) : (
-                            <img id="loadingIcon" src="./assets/images/loading.gif" alt="loading"/>
+                            <img id="loadingIcon" src="./assets/images/loading-wheel.gif" alt="loading"/>
```
Comment (TheGuardingDark) [inline client/src/pages/Profile/Profile.js]: Did you want a different loading gif?

File: client/src/pages/Profile/Profile.js [hunk 2]
```diff
@@ -22,14 +29,16 @@ class Profile extends Component {
             if (user.data.loggedIn) {
                 this.setState({
                     loggedIn: true,
-                    user: user.data.user
+                    user: user.data.user,
```
Comment (TheGuardingDark) [inline client/src/pages/Profile/Profile.js]: should there be a comma here?

File: client/src/pages/Profile/Profile.js (hunk 1 above)
Comment (wynnc) [reply client/src/pages/Profile/Profile.js]: I'm not sure what they were pointing at. it was giving an error and that loading-wheel image was the only one in the folder. we will let them figure it out.

File: client/src/pages/Home/Home.js [hunk 3]
```diff
@@ -30,22 +30,22 @@ class Home extends Component {
   render() {
     return (
       <>
-        <section class="landing">
-          <div class="landing-inner">
+        <section className="landing">
+          <div className="landing-inner">
             <Container>
-              <Row centered>
+              <Row>
```
Comment (TheGuardingDark) [inline client/src/pages/Home/Home.js]: do we need the centered modifier?

File: client/src/pages/Profile/Profile.js (hunk 2 above)
Comment (wynnc) [reply client/src/pages/Profile/Profile.js]: it is ok. it is not causing any harm.

File: client/src/pages/Home/Home.js (hunk 3 above)
Comment (wynnc) [reply client/src/pages/Home/Home.js]: this was giving an error. so I removed it. it did not change the styling on the page so I'm not sure what the desired outcome was for this particular styling.

**Pull request merged**